dealership/vehicles.py

```diff
@@ -1,15 +1,50 @@
 class Vehicle(object):
+    sale_multiplier = None
```

Comment:
Are these attributes really needed here? Since you have it declared down there for each sub-class I don't think you need them here, unless you are using them somewhere else and I can't see it.